### PR TITLE
Add templates

### PR DESCRIPTION
**Pull request by Cosifne**
I just follow the structure of other projects in this repo to create the vsixmanifest and some other file. Not sure it is correct because I can't test this repo on my machine. Needs @singhsarab 's help

@shyamnamboodiripad  we can first make sure the content of the templates fits our needs.
Some updates:
Currently we don't want to this WinAppDriver test template immediately because it can just work on .NET framework.
Also, as for the some nuget packages for this templates here, it can't use the newest version. I check the email thread with the WinAppDriver team. They say it is because these version it the most stable one they have tested.
Overall,  many problems about this template

### CODE REVIEW

File: Templates/CSharp/MSTestWinAppDriverCSharp/ProjectTemplates/CSharp/WinAppDriver/ScenarioTest.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+namespace $safeprojectname$
+{
+    [TestClass]
+    public class ScenarioTest : TestSession
+    {
+        [TestMethod]
+        public void Test()
+        {
+            // Add the test logic
```
Comment (Cosifne) [inline Templates/CSharp/MSTestWinAppDriverCSharp/ProjectTemplates/CSharp/WinAppDriver/ScenarioTest.cs]: Should we put some sample statement here ? Or just put these comments? @shyamnamboodiripad

File: Templates/CSharp/MSTestWinAppDriverCSharp/ProjectTemplates/CSharp/WinAppDriver/TestSession.cs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+using OpenQA.Selenium.Appium.Windows;
+using OpenQA.Selenium.Remote;
+using OpenQA.Selenium;
+using System;
+
+namespace $safeprojectname$
+{
+    // In order to run test, please download and install Win App Driver from https://github.com/Microsoft/WinAppDriver/releases.
+    // Then run it in background, default it listens to 127.0.0.1:4723
+    // For more information, see https://github.com/Microsoft/WinAppDriver
```
Comment (Cosifne) [inline Templates/CSharp/MSTestWinAppDriverCSharp/ProjectTemplates/CSharp/WinAppDriver/TestSession.cs]: We may provide more information here? @shyamnamboodiripad

File: Templates/CSharp/MSTestWinAppDriverCSharp/ProjectTemplates/CSharp/WinAppDriver/TestSession.cs [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    public class TestSession
+    {
+        // Win App Driver runs on localhost:4723 by default
+        protected const string WindowsApplicationDriverUrl = "http://127.0.0.1:4723";
+        // Provide the app id of your application
+        private const string ApplicationID = @"Your Application ID";
```
Comment (Cosifne) [inline Templates/CSharp/MSTestWinAppDriverCSharp/ProjectTemplates/CSharp/WinAppDriver/TestSession.cs]: Is it clearly enough to the user they need to enter their application ID? @shyamnamboodiripad

**Comment by singhsarab** [comment]
@Cosifne Is this still relevant ?

**Comment by Cosifne** [comment]
> 
> 
> @Cosifne Is this still relevant ?

I remember that we temporary decide to pause adding this because of some shipping problem. I will abondon this one

**Pull request closed without merging**